config/puma.rb

```diff
@@ -0,0 +1,56 @@
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
Unicorn used to be the default back when this was first created so it makes sense. Lets concentrate on getting the rater sorted and running and then do the puma as a separate pull request - I have a feeling you might need to write some configuration for it? I've only used it once before and the person that handed me the project had already configured it.